pkg/hyper/client.go

```diff
@@ -189,14 +191,48 @@ func (c *Client) GetContainerInfo(container string) (*types.ContainerInfo, error
 	return cinfo.ContainerInfo, nil
 }
 
+// StartContainer starts a hyper container
+func (c *Client) StartContainer(containerID string) error {
+	isRunning, err := isContainerRunning(c, containerID)
+	if err != nil {
+		return err
+	}
+	if isRunning {
+		glog.V(3).Infof("Container %q is already running, skip", containerID)
+		return nil
+	}
+
+	ctx, cancel := getContextWithTimeout(hyperContextTimeout)
+	defer cancel()
+
+	_, err = c.client.ContainerStart(ctx, &types.ContainerStartRequest{ContainerId: containerID})
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// RemoveContainer removes a hyper container
+func (c *Client) RemoveContainer(containerID string) error {
+	ctx, cancel := getContextWithTimeout(hyperContextTimeout)
+	defer cancel()
+
+	_, err := c.client.ContainerRemove(ctx, &types.ContainerRemoveRequest{ContainerId: containerID})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
 // StopContainer stops a hyper container
 func (c *Client) StopContainer(containerID string, timeout int64) error {
 	if timeout <= 0 {
 		return fmt.Errorf("Timeout can not be %d, it must be greater than zero.", timeout)
 	}
 
 	// do checks about container status
-	err := c.CheckIfContainerRunning(containerID)
+	err := ensureContainerRunning(c, containerID)
 	if err != nil {
```

Comment:
Should also handle stopped container here, return nil if the container has already been stopped